variables.tf

```diff
@@ -102,6 +116,11 @@ variable "enabled_cluster_log_types" {
     default = []
 }
 
+variable "monitoring" {
```

Comment:
We can use a more descriptive variable name here, for example: var.enable_detailed_monitoring

Reply:
Done